ykman-gui/qml/YubiKey.qml

```diff
@@ -26,6 +26,8 @@ Python {
     property bool yubikeyModuleLoaded: false
     property bool yubikeyReady: false
     property var queue: []
+    property var piv
```

Comment:
These should probably live in the PivView.qml for now, or we should move out all data variables from the views.

Reply:
Agreed offline to hold off on this for now.